network-api/networkapi/templates/fragments/buyersguide/pni_sort_dropdown.html

```diff
@@ -0,0 +1,60 @@
+{% load static wagtailcore_tags wagtailimages_tags i18n %}
+<ul 
```

Comment:
The guide I used did lists so I kept the convention to make it easier to follow when doing to the JS part. I would pick Detail if it was a bit more simple like the ones we use for our accordion dropdown

Reply:
Can you expand on why `details` would only work for simple cases? Also, do you have a link to the guide you've used -- maybe that provides some context that is  not obvious to me here.

Reply:
https://dev.to/emmabostian/creating-a-custom-accessible-drop-down-3gmo

Here is the guide I followed

Reply:
I think detail would work but I would probably require more then just removing the JS parts related to toggling. It does require different CSS overrides that `li/ul`.

I also do not have knowledge about how `details/summary` screenreader/accessibiliity requirements or how to actually add aria-tags properly on it. If I want to use details for something outside of accordion usage, I would like to do a double check for aria-related tags.

I would like this to be a spike ticket since we would probably need to do a11y research for both the category dropdown and sort dropdown.

There are still a few accessibility bugs with detail-summary for example:
https://a11ysupport.io/tests/tech__html__details-summary#assertion-html-h1-6_elements-convey_role_and_name-

Reply:
Thanks for the guide. 

Looks like the accessibility issue are related to using a heading inside the summary. I don't think that's something we would want to do here.